.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Set default behaviour for normalizing line endings to LF
+* text=auto eol=lf
```

Comment:
I wonder if there is a binary file extension (even if added to repo in future) that would get mangled due to this :thinking: 

Would it make to sense to specify extensions where we want the line-endings changed vs doing it across the board?

Reply:
Purely wondering, if you think there is no action needed we can leave it as-is

Reply:
That's a good question. I would _hope_ that `git`/GitHub are smart enough to handle this correctly, but it's a valid concern. Although, to play devil's advocate, I could also argue that if we add new text file formats not included in `.gitattributes`, we wouldn't be able to enforce LF line endings for those without also updating `.gitattributes`.

In either case, we would need to update `.gitattributes` accordingly, so I feel like we could go either way.

> I wonder if there is a binary file extension (even if added to repo in future) that would get mangled due to this

I would hope that if this happens, we can catch it before the offending PR is merged.